blocks/core/ff_module/ff_module-form-marking-controls/ff_module-form-marking-controls.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var React = require('react');
+var ReactDOM = require('react-dom');
+
+var Button = require('../ff_module-button/ff_module-button');
+var CheckableList = require('../ff_module-form-checkable-list/ff_module-form-checkable-list');
+var ControlBar = require('../../ff_container/ff_container-control-bar/ff_container-control-bar');
+
+module.exports = React.createClass({
+	displayName: 'MarkingControls',
+	generateClass: function(base, props) {
+		var classNames = [];
+		props = props || {};
+		classNames.push(base);
+		if (!!props.modifier) classNames.push(base + '--' + props.modifier);
+		if (!!props.classes) classNames.push(props.classes);
+		if (!!props.className) classNames.push(props.className);
+		return classNames.join(' ');
+	},
+	render: function() {
+		return <div className={this.generateClass('ff_module-form-marking-controls', this.props.modifier)}>
+			<ControlBar modifier={this.props.controlBar.modifier}>
```

Comment:
Again, I'd change this to <ControlBar {...this.props.controlBar}> so you can pass all the props that ControlBar uses without needing to redefine every time :)